packages/members/theme-dropin/src/index.js

```diff
@@ -40,7 +43,7 @@ DomReady(function () {
     if (blogUrl && postId) {
         init(blogUrl, postId);
     } else {
-        GhostMembersLayer2.init({reload: true});
+        GhostMembersLayer2.init({reload: true, blogUrl});
```

Comment:
blogUrl ccan be undefined - we should do somethign about that - no necessary now tho